ci-operator/config/openshift/release/openshift-release-master__okd-4.15.yaml

```diff
@@ -32,6 +32,13 @@ base_images:
     namespace: origin
     tag: upi-installer
 releases:
+  initial:
+    prerelease:
+      product: okd
+      version_bounds:
+        lower: 4.13.0-0
```

Comment:
```suggestion
        lower: 4.14.0-0
```

Reply:
This shouldn't be necessary, as it'll always pick the latest one from https://origin-release.ci.openshift.org/#4-stable now